### ISSUE TITLE
Local polling via MQTT to Home Assistant

### ISSUE THREAD

**Issue by tmh88** [cdpuk/ha-bestway#67]
**Describe the solution you'd like**
It would be nice if we could have local polling via MQTT to home assistant.

Reading the information, it looks like some work has been done already.
How can we as a community help the dev to implement this?

I'm happy to help with the development

**Comment by cdpuk** [comment]
Is this different to #28 (which proposes a WebSocket connection)? I wasn't aware of any MQTT support - could you link to a reference?

**Comment by lenard-a** [comment]
I think @tmh88 was referring to this part in the Improvement ideas section of the readme file:

> We know the spa talks directly to the cloud using MQTT. Traffic captures against the Android app appeared to show only HTTPS traffic.

**Comment by cdpuk** [comment]
Ah I see. I made that comment quite a while ago - I even wonder whether I actually meant WebSocket when I wrote that. I suggest we keep the discussion about alternative comms methods in #28. Clearly a change to some other more reliable and ideally local communication method would be great, however it requires significant research and a rewrite of a large portion of the integration.  I certainly don't have time for that right now, but happy to see discussion on the topic.
